onnxruntime/contrib_ops/rocm/bert/timer.cc

```diff
@@ -1,7 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think this file should be moved to `rocm` directory, since it is not specific to bert